src/compas/datastructures/network/planarity_.py

```diff
@@ -1,326 +0,0 @@
-from __future__ import print_function
```

Comment:
Same comment on this file, the file is deleted, but the functions are still listed on the API, so, are they coming from somewhere else?

Reply:
yes they are in `planarity.py`. the file was just still on my system and therefore in the PR...